tox.ini

```diff
@@ -17,7 +17,7 @@ commands = coverage run --parallel -m pytest {posargs}
 [testenv:flake8]
 basepython = python3.5
 skip_install = true
-deps = flake8
+deps = flake8~=3.2.0
```

Comment:
Please don’t do this; it’s OK if stuff fails on releases. :)  Otherwise thanks, will merge once this is gone.